cpp/server/ct-server.cc

```diff
@@ -222,10 +231,30 @@ int main(int argc, char* argv[]) {
   // For now, run with a dedicated thread pool as the executor for our
   // consistent store to avoid the possibility of DoS through thread starvation
   // via HTTP.
-  ThreadPool etcd_pool(2);
-  EtcdConsistentStore<LoggedCertificate> consistent_store(&etcd_pool,
+  ThreadPool internal_pool(4);
+  EtcdConsistentStore<LoggedCertificate> consistent_store(&internal_pool,
                                                           &etcd_client,
                                                           "/root", node_id);
+  // Put a sensible single-node config into FakeEtcd. For a real clustered log
+  // we'd expect a ClusterConfig already to be present within etcd as part of
+  // the provisioning of the log.
+  {
+    ct::ClusterConfig config;
+    config.set_minimum_serving_nodes(1);
+    config.set_minimum_serving_fraction(1);
+    LOG(INFO) << "Setting default single-node ClusterConfig:\n"
+              << config.DebugString();
+    consistent_store.SetClusterConfig(config);
```

Comment:
This kinds of buries the defaults in the code, though... Having the defaults in the protobuf makes them easy to find and read (you could add some comments explaining the settings there, which could serve as a reference), it keeps it all in one place, and makes it impossible for a code path to not have a setting (if the setting exists, it will absolutely have a default, and not ```!has_that_setting()```), even as they are added (would the defaults be explicitly set by the initial config tool, or when the protobuf is read? if the first, what when a setting is added, and it's not in the existing config?).

It also makes the config tool potentially really simple, taking an ASCII protobuf config file, say.